msdk-io/msdk-io-mzml2/src/main/java/io/github/msdk/io/mzml2/MzMLCV.java

```diff
@@ -0,0 +1,125 @@
+/*
+ * (C) Copyright 2015-2017 by MSDK Development Team
+ *
+ * This software is dual-licensed under either
+ *
+ * (a) the terms of the GNU Lesser General Public License version 2.1 as published by the Free
+ * Software Foundation
+ *
+ * or (per the licensee's choosing)
+ *
+ * (b) the terms of the Eclipse Public License v1.0 as published by the Eclipse Foundation.
+ */
+
+package io.github.msdk.io.mzml2;
+
+import uk.ac.ebi.jmzml.model.mzml.CVParam;
+
+/**
+ * Controlled vocabulary (CV) values for mzML files.
+ * 
+ * @see <a href=
+ *      "http://psidev.cvs.sourceforge.net/viewvc/psidev/psi/psi-ms/mzML/controlledVocabulary/psi-ms.obo">
+ *      Official CV specification</a>
+ * 
+ */
+class MzMLCV {
+
+	static enum MzMLCompressionType {
+		ZLIB("MS:1000574"); // UNCOMPRESSED, MSNUMPRESS
+							// Unable to find corresponding Accession Codes
```

Comment:
Check the official controlled vocabulary documentation: https://raw.githubusercontent.com/HUPO-PSI/psi-ms-CV/master/psi-ms.obo
It seems there are 6 different types of numpress compression CV terms